src/Field/CoordinateReferenceSystemCombo/CoordinateReferenceSystemCombo.spec.jsx

```diff
@@ -66,11 +65,13 @@ describe('<CoordinateReferenceSystemCombo />', () => {
   describe('#onFetchError', () => {
     it('logs error message', () => {
       const wrapper = TestUtil.mountComponent(CoordinateReferenceSystemCombo);
-      const loggerSpy = sinon.spy(Logger, 'error');
+      const loggerSpy = jest.spyOn(Logger, 'error');
       wrapper.instance().onFetchError('Peter');
-      expect(loggerSpy.calledOnce).toBeTruthy();
-      expect(loggerSpy.calledWith('Error while requesting in CoordinateReferenceSystemCombo: Peter')).toBeTruthy();
-      loggerSpy.restore();
+      expect(loggerSpy).toHaveBeenCalled();
+      expect(loggerSpy.mock.calls[0][0]).toBe('Error while requesting in CoordinateReferenceSystemCombo: Peter');
```

Comment:
We might better use `.toHaveBeenCalledWith`  https://facebook.github.io/jest/docs/en/expect.html#tohavebeencalledwith